tests/src/Kernel/CoolPreviewFormatterTest.php

```diff
@@ -89,15 +90,11 @@ public function testViewElements(): void {
       $field_name => 1,
     ]);
     $entity->save();
+    $build = $test_display->build($entity)[$field_name];
+    // Default formatter is applied.
     $this->assertSame(
-      [
-        '#cache' => [
-          'contexts' => [],
-          'tags' => [],
-          'max-age' => Cache::PERMANENT,
-        ],
-      ],
-      $test_display->build($entity)[$field_name]
+      'file_default',
+      $test_display->build($entity)[$field_name]['#formatter']
```

Comment:
Now we are calling ->build() twice.

Reply:
```suggestion
      $build['#formatter']
```
or drop the other call to `->build()` above.